x-pack/plugins/security_solution/public/app/home/index.tsx

```diff
@@ -50,7 +50,7 @@ const HomePageComponent: React.FC<HomePageProps> = ({
     <SecuritySolutionAppWrapper className="kbnAppWrapper">
       <GlobalHeader setHeaderActionMenu={setHeaderActionMenu} />
       <DragDropContextWrapper browserFields={browserFields}>
-        <UseUrlState indexPattern={indexPattern} navTabs={navTabs} />
+        <UseUrlState indexPattern={indexPattern} navTabs={navTabs} currentPath={pathname} />
```

Comment:
I know, that there is no unit test file for this component, do you think it will be possible to add something in this PR or better open a follow up issue for the unit test coverage around this?

Reply:
Thanks for the feedback. I want to add unit & integration testing around this in a follow up PR, prior to doing the refactoring of this entire component.